blockchain/validate.go

```diff
@@ -2446,8 +2452,8 @@ func checkRevocationInputs(tx *dcrutil.Tx, txHeight int64, view *UtxoViewpoint,
 		return ruleError(ErrImmatureTicketSpend, str)
 	}
 
-	ticketOuts := ConvertUtxosToMinimalOutputs(ticketUtxo)
-	if len(ticketOuts) == 0 {
+	ticketOuts := ticketUtxo.TicketMinimalOutputs()
+	if ticketOuts == nil {
```

Comment:
Updated, though I don't think the case of it having a zero length and not being nil is possible, since if there were no outputs then `ticketUtxo` itself would be nil.

Reply:
Yeah, I agree with the current code that is true, but this is an assertion to try and catch bugs as things evolve.  I could see a case later were a utxo entry is reused in the cache and the assumption gets violated where it forgets to nil it out and instead ends up with a slice that has been cleared.